### DISCUSSION TITLE
[Tailwind UI]: How would you handle alerts on a light grey background?

### DISCUSSION THREAD

**Discussion by CodeAndWeb** [tailwindlabs/tailwindcss#2478] [Help]
It would be nice to have alerts in a card-layout on light gray background. The current alerts look great on white - but don't have enough contrast on light-grey:

<img width="1038" alt="2020-09-07_09-22-54" src="https://user-images.githubusercontent.com/716009/92359618-daf9b480-f0eb-11ea-9924-942cc1d3855a.png">

**Comment by alexanderbluhm** [comment]
Quite difficult to come up with a good solution 🤔 
I tested a few things came to my mind.
![tui_alert](https://user-images.githubusercontent.com/6071634/92483127-bf280880-f1e8-11ea-92c0-f895a934c297.png)

The red background is too much for me. The border and outline variant look "ok" for me.
Maybe Steve comes up with a nicer idea :)

**Comment by CodeAndWeb** [comment]
I'd not use the shadow-outline... it's reserved for the focus indicator.
Border indeed looks "ok". Going to test it in combination with the shadow....
